workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs

```diff
@@ -85,6 +109,52 @@ public static EntityTemplate Player(string workerId, byte[] args)
                 RegenPauseTime = 0,
             };
 
+            var playerInterest = new ComponentInterest
+            {
+                Queries = new List<ComponentInterest.Query>
+                {
+                    new ComponentInterest.Query
+                    {
+                        Constraint = new ComponentInterest.QueryConstraint
+                        {
+                            ComponentConstraint = PlayerState.ComponentId,
+                            AndConstraint = new List<ComponentInterest.QueryConstraint>(),
+                            OrConstraint = new List<ComponentInterest.QueryConstraint>(),
+                        },
+                        FullSnapshotResult = true,
+                        ResultComponentId = new List<uint>(),
+                    },
+                    new ComponentInterest.Query
+                    {
+                        Constraint = new ComponentInterest.QueryConstraint
+                        {
+                            ComponentConstraint = Session.ComponentId,
+                            AndConstraint = new List<ComponentInterest.QueryConstraint>(),
+                            OrConstraint = new List<ComponentInterest.QueryConstraint>(),
+                        },
+                        FullSnapshotResult = true,
+                        ResultComponentId = new List<uint>(),
+                    }
+                }
+            };
+
+            var interestComponent = new Interest.Snapshot
+            {
+                ComponentInterest = new Dictionary<uint, ComponentInterest>
+                {
+                    { ClientMovement.ComponentId, playerInterest },
+                },
+            };
```

Comment:
🙏  great use-case for the QBI helper module :D 

something along the lines of:

```csharp
var playerStateQuery = InterestQuery.Query(Constraint.Component<PlayerState.Component>());
var sessionQuery = InterestQuery.Query(Constraint.Component<Session.Component>());

var interestTemplate = InterestTemplate.Create()
    .AddQueries<ClientMovement.Component>(playerStateQuery, sessionQuery);

var interestComponent = interestTemplate.ToSnapshot();
```

Reply:
perfect! :D